Cabal-syntax/src/Distribution/Utils/Path.hs

```diff
@@ -53,8 +55,6 @@ module Distribution.Utils.Path
   , interpretSymbolicPathAbsolute
 
     -- ** General filepath API
-  , (</>)
-  , (<.>)
```

Comment:
Why remove them?

Reply:
They are exported via `(..)`.

Reply:
@Bodigrim: apparently, the comment needs to be manually resolved for merging to be legal, after @kazu-yamamoto sets the merge label.